### DISCUSSION TITLE
Roadmap to install GeoWrangler via Conda

### DISCUSSION THREAD

**Discussion by gabosantos** [thinkingmachines/geowrangler#204] [Enhancements Requests]
Hi TM,

Currently, the only way to install GeoWrangler is via pip.  Are there any plans of putting this on Anaconda?

Thanks
Gabriel

**Comment by jtmiclat** [comment]
Hi @gabosantos we currently don't have plans in supporting conda and are only supporting pypi for now. Conda currently supports interoperability with pip/pypi https://docs.conda.io/projects/conda/en/latest/user-guide/configuration/pip-interoperability.html  which might be helpful

But let us know if you are hitting roadblocks with using geowrangler in conda env so we can revisit if it prevents our users from accessing the tool!

**Reply by gabosantos** [reply]
Thank you, @jtmiclat. So far, no issues encountered using this path.
